project-repo/templates/common/.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,83 @@
+name: Request a feature
+description: Request a feature
```

Comment:
Shouldn’t that be `Suggest new functionality for this project.` as in the previous scaffolding?